Detectors/TPC/simulation/include/TPCSimulation/DigitContainer.h

```diff
@@ -57,13 +58,14 @@ class DigitContainer{
     /// \param charge Charge of the digit
     void addDigit(size_t hitID, int cru, int timeBin, int row, int pad, float charge);
 
-    /// Fill output TClonesArray
+    /// Fill output vector
     /// \param output Output container
     /// \param mcTruth MC Truth container
     /// \param debug Optional debug output container
     /// \param eventTime time stamp of the event
     /// \param isContinuous Switch for continuous readout
-    void fillOutputContainer(TClonesArray *output, o2::dataformats::MCTruthContainer<o2::MCCompLabel> &mcTruth, TClonesArray *debug, int eventTime=0, bool isContinuous=true);
+    void fillOutputContainer(std::vector<o2::TPC::Digit> *output, o2::dataformats::MCTruthContainer<o2::MCCompLabel> &mcTruth,
```

Comment:
same question here about the use of a pointer vs. reference on the first argument